.travis.yml

```diff
@@ -43,13 +43,13 @@ services:
 script:
   - npm run jscs-enforce
   - npm run checkdb
-  - npm run test-travis
+  - travis_retry npm run test-travis
 
 before_install:
-  - npm install -g npm@^5.7.0
+  - travis_retry npm install -g npm@^5.7.0
 
 install:
-  - npm ci
+  - travis_retry npm ci
```

Comment:
Do we need the retry for before_install and install?

Reply:
The "typical" use case for this feature is to retry if you get network timeouts when trying to download dependencies, so I figured once I was retrying our tests, I may as well also protect against problems with npm component installation